packages/web-components/@groupby/elements-search-box/test/unit/search-box.test.ts

```diff
@@ -146,37 +146,59 @@ describe('SearchBox Component', () => {
     });
   });
 
-  describe('updateText', () => {
+  describe('updateSearch', () => {
     const term = 'inputText';
     const group = 'some-group';
     let updateSearchTermValue;
+    let emitSearchEvent;
 
     beforeEach(() => {
       updateSearchTermValue = stub(searchbox, 'updateSearchTermValue');
+      emitSearchEvent = stub(searchbox, 'emitSearchEvent');
     });
 
     it('should update the value property with data from the event when the event group matches the component group', () => {
       const inputEvent = new CustomEvent('some-test-type', { detail: { term, group } });
       searchbox.group = group;
 
-      searchbox.updateText(inputEvent);
+      searchbox.updateSearch(inputEvent);
 
       expect(updateSearchTermValue).to.be.calledWith(term);
     });
 
-    it('should not update the value property with data from the event when the group in the component and the event do not match', () => {
+    it('should emit a search request if event requests it and group matches', () => {
```

Comment:
maybe reword to something like '...if event payload has search property set to true...'

Reply:
we didn't make it something much longer like that because we were trying to reduce line lengths for test descriptions. This also has the benefit of not being too formal while having the details in the test.